### ISSUE TITLE
[metaltracker] Login failed: No form found on https://en.metal-tracker.com/user/login.html using form selector form (Config)

### ISSUE THREAD

**Issue by mrelcee** [Jackett/Jackett#4912]
jackett fails to login to metal-tracker

An error occured while updating this indexer
Login failed: No form found on https://en.metal-tracker.com/user/login.html using form selector form

jacket 0.11.15.0
Mono 5.10.1.57  (FreeBSD 11.2)

**Comment by garfield69** [comment]
cannot replicate your issue, works for me on jackett 0.11.60 win10.

try upgrading to 0.11.60

is that does not work, try deleting your indexer and adding it afresh.

if that does not work then please provide the **full enhanced log** file. 
1. scroll down to the bottom of the Jackett Dashboard and *tick* the *enhanced Logging* checkbox
2. scroll up a bit and *click* on the *apply server settings* button
3. repeat the test
4. find the *log.txt* file (linux `~/.config/Jackett/`, windows `%ProgramData%\Jackett`)
5. edit it with a plain text editor, redacting any personal details, usernames, passwords, passkeys, hashes etc.
6. save, then drag-drop it here for us to take a look at.
Thanks.

**Comment by tijrt** [comment]
I too had this exact error for metal-trackers for weeks, but resolved with a little trial error. Unfortunately, I doubt my logs would help OP. 

Build - Windows Server 2012 with Jacket v. .10.890 running as a service but accessing via separate workstation through Chrome. 

I logged into my server, restarted the service and connected to the localhost IP + port and was able to add the tracker using Chrome. Once I did this, returning back to my workstation and refreshing the page the tracker shows up and is working again. 

Hope it's some help!
